src/bidiMapper/domains/context/CdpTarget.ts

```diff
@@ -135,9 +143,9 @@ export class CdpTarget {
           ignore: this.#acceptInsecureCerts,
         }),
         // TODO: enable Network domain for OOPiF targets.
-        enabledNetwork
-          ? this.#cdpClient.sendCommand('Network.enable')
-          : undefined,
+        this.toggleNetworkIfNeeded(enabledNetwork).then(async () => {
+          return await this.toggleFetchIfNeeded();
```

Comment:
should not we enable the fetch at the same time as the network? I think this might be flaky since the target will unblock before the fetch is configured (if needed)

Reply:
Agree, let me change it, also now I see there was a small bug, that we never set the `networkDomainEnabled` var in call.

Reply:
Done.